src/main/java/duke/command/ExitCommand.java

```diff
@@ -0,0 +1,20 @@
+package duke.command;
+
+import duke.common.DukeString;
+import duke.task.TaskList;
+
+/**
+ * Represents a command that tells the parser to exit.
+ */
+public class ExitCommand implements Command {
+    /**
```

Comment:
Perhaps you could standardize if you would like to add a blank line after declaring class or not? I have noticed inconsistencies of blank spaces after class declaration in several other places too.